config/kinoite-packages.yml

```diff
@@ -7,34 +7,14 @@ remove:
   - plasma-welcome
   - kfind
   - kcharselect
-  - kde-connect
-  - kde-connect-libs
-  - kdeconnectd
-  - plasma-discover
   - plasma-discover-notifier
   - kf5-akonadi-server
   - kf5-akonadi-server-mysql
-  - kde-partitionmanager
-  - kwalletmanager5
-  - plasma-vault
-  - flatpak
```

Comment:
Thanks for your PR. Most of it is good, but please make the following changes:

- Don't add flatpak back. It won't work at all yet since unprivileged user namespaces are disabled, so there's no reason to add it back until we have bubblewrap-suid installed.
- Don't add cups or anything printer-related back. Let's have a separate issue for discussing that. CUPS exposes attack surface and it was removed as a security improvement, not for "debloating", so it's not related to this PR

Reply:
Also, please combine all your commits into a single commit

Reply:
done, I am using Github in the Browser as I constantly fail to set it up permanently, so I cannot squash for some reason

Reply:
please combine your commits, if you need help setting it up let me know what issue you're facing